lib/msal-angular/src/msal-guard.service.ts

```diff
@@ -90,4 +90,8 @@ export class MsalGuard implements CanActivate {
         return (typeof str === "undefined" || !str || 0 === str.length);
     };
 
+    isObjectEmpty(obj: Object) {
```

Comment:
should this be in a util?

Reply:
Ideally yes but there is no util currently for angular and we can move a bunch of functionality when we get there. Did not want to introduce it as a part of this PR.